eZ/Publish/API/Repository/Tests/FieldType/RelationIntegrationTest.php

```diff
@@ -109,7 +113,7 @@ public function getValidatorSchema()
      */
     public function getValidFieldSettings()
     {
-        return array('selectionMethod' => 0, 'selectionRoot' => '1');
+        return array('selectionMethod' => 0, 'selectionRoot' => '1', 'selectionContentTypes' => array());
```

Comment:
> I used the old syntax cause it is used everywhere in the file, well every file I touched for these changes... didnt want to mix the style. If I fix in the whole file then the diff becomes messy, if I fix only for added lines then the source will be ugly I think.

Well, we need our code to evolve (a.k.a not making a revolution), so we decided that for modified and new code we'll use short array syntax. So if we modified a big multidimensional array, we'd change the whole thing. Could be as a separate commit. However we don't change unrelated and unmodified blocks so diff won't get too messy. Look at this as a compromise :)